tests/load_cache.test/runit

```diff
@@ -0,0 +1,272 @@
+#!/usr/bin/env bash
```

Comment:
A lot of code here is useful and not specific to this test (eg bounce_cluster).  Maybe we should move it to a file in tests and source it from here?